### PR TITLE
Update readme to include steps to install latest NATS manager and CR

### PR DESCRIPTION
**Pull request by marcobebway**
**Description**

Update readme to include steps to install latest NATS manager and CR.

### CODE REVIEW

**Review by mfaizanse — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -16,6 +16,20 @@ It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controlle
 
 This project is scaffolded using [Kubebuilder](https://book.kubebuilder.io), and all the Kubebuilder [`makefile` helpers](https://book.kubebuilder.io/reference/makefile-helpers.html) can be used.
 
+## Install
+
+Run the following command to install the latest version of the NATS manager on your cluster:
```
Comment (NHingerl) [inline README.md:21]:
```suggestion
1. To install the latest version of the NATS manager on your cluster, run:
```
"First purpose, then instruction" makes it easier to detect what this is about (especially in contrast to the 2nd instruction).

File: README.md [hunk 2]
```diff
@@ -16,6 +16,20 @@ It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controlle
...
+
+```bash
+kubectl apply -f https://github.com/kyma-project/nats-manager/releases/latest/download/nats-manager.yaml
+```
+
+Run the following command to install the latest version of the default NATS CR on your cluster:
```
Comment (NHingerl) [inline README.md:27]:
```suggestion
2. To install the latest version of the default NATS CR on your cluster, run:
```

File: README.md [hunk 3]
```diff
@@ -16,6 +16,20 @@ It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controlle
 
 This project is scaffolded using [Kubebuilder](https://book.kubebuilder.io), and all the Kubebuilder [`makefile` helpers](https://book.kubebuilder.io/reference/makefile-helpers.html) can be used.
 
+## Install
+
```
Comment (NHingerl) [inline README.md:20]: Are these instructions alternatives to each other, or subsequent steps?
I suspect they are both required, not alternatives. 
In that case, use a numbered list and add indentations to the code blocks so they're properly aligned below the respective step.

File: README.md [hunk 4]
```diff
@@ -16,6 +16,20 @@ It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controlle
...
+Run the following command to install the latest version of the NATS manager on your cluster:
+
+```bash
```
Comment (NHingerl) [inline README.md:23]: add 3 blank spaces to each line of the code block

File: README.md [hunk 5]
```diff
@@ -16,6 +16,20 @@ It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controlle
...
+kubectl apply -f https://github.com/kyma-project/nats-manager/releases/latest/download/nats-manager.yaml
+```
+
+Run the following command to install the latest version of the default NATS CR on your cluster:
+
+```bash
```
Comment (NHingerl) [inline README.md:29]: add 3 blank spaces to each line of the code block

**Review by NHingerl — COMMENTED** [review]
Remember that on the OS website, doc [Install Kyma with a Module](https://kyma-project.io/#/02-get-started/08-install-uninstall-upgrade-kyma-module), will also need these commands when we're GA.

File: README.md (hunk 3 above)
Comment (marcobebway) [reply README.md:20]: They are required steps steps.

File: README.md (hunk 4 above)
Comment (marcobebway) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (marcobebway) [reply README.md]: Done

**Review by NHingerl — APPROVED** [review]

**Pull request merged**